pkg/apis/workload/v1alpha1/synctarget_types.go

```diff
@@ -71,8 +71,8 @@ type SyncTargetSpec struct {
 	// will be selected to deploy the workload only when the resource schema on the SyncTarget is compatible
 	// with the resource schema included in the exports.
 	// If it is not set, the kubernetes export in the same workspace will be used by default.
-	// +kubebuilder:default={{workspace: {exportName: kubernetes}}}
-	SupportedAPIExports []apisv1alpha1.ExportReference `json:"supportedAPIExports,omitempty"`
+	// +kubebuilder:default={{exportName: kubernetes}}
+	SupportedAPIExports []apisv1alpha1.WorkspaceExportReference `json:"supportedAPIExports,omitempty"`
```

Comment:
actually, wasn't `ExportReference` correct? We might eventually (with service catalog features) have another way to refrence exports.

Reply:
of course we need an extension to the yaml patch for ExportReference, i.e. the OneOf that at least one of {Workspace} is set.